tensorflow/contrib/verbs/rdma_rendezvous_mgr.cc

```diff
@@ -74,8 +77,33 @@ void RdmaRemoteRendezvous::RecvFromRemoteAsync(
     return;
   }
 
+  // Type-specialized logging for this method.
+  WorkerCacheLogger* logger = rdma_mgr_->GetLogger();
+
+  bool logging_active = logger->LoggingActive();
+  DoneCallback wrapper_done;
+  const DoneCallback* cb_to_use;
+  if(!logging_active) { 
+    cb_to_use = &done; //No additional work to do, so just use done directly
+  } else {
+     int64 step_id = step_id_;
+     wrapper_done = [this, logger, step_id, parsed, done, start_usec] (const Status& s, const Args& send_args, const Args& recv_args,
+  	                                                               const Tensor& recv_tensor, const bool is_dead) {
+      
+      if (logger->LoggingActive()) {
```

Comment:
Is this line necessary? I think `else` in line 88 checks this condition.

Reply:
This is callback function for an asynchronous call, so I think it is for a recheck the logging activation.